### ISSUE TITLE
More details needed

### ISSUE THREAD

**Issue by danmyhre** [MicrosoftDocs/memdocs#2142]
Please add some details about what isn't supported on cloud PCs for MEM compliance policies. For example, secure boot and code isolation.

Can you also clarify what this means? 
"Windows health attestation settings cannot be validated on virtual machines including Cloud PCs."

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 05938165-8748-6557-6524-1f47f6ed2383
* Version Independent ID: ee7f960a-6e97-9c4a-b9ea-2205b3af9d47
* Content: [Security overview for Windows 365](https://docs.microsoft.com/en-us/windows-365/enterprise/security-guidelines)
* Content Source: [windows-365/enterprise/security-guidelines.md](https://github.com/MicrosoftDocs/memdocs/blob/master/windows-365/enterprise/security-guidelines.md)
* Service: **cloudpc**
* Sub-service: ****
* Technology: ****
* GitHub Login: @ErikjeMS
* Microsoft Alias: **erikje**

**Comment by ErikjeMS** [comment]
Investigating with PM
37830914

**Comment by ErikjeMS** [comment]
@danmyhre We've replaced that line in the doc with the following note: 
Cloud PCs aren't created with Secure Boot enabled and Cloud PCs don't support BitLocker. We recommend excluding these two settings from compliance policies targeting Cloud PCs.
Also, we've added a FAQ item here:
https://docs.microsoft.com/windows-365/enterprise/known-issues-enterprise#cloud-pc-reports-as-not-compliant-for-compliance-policy ?
If this doesn't answer your question, let us know. Thanks!
